src/common/copyOfHDSLink/Link.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+  This is a copy of HDS Link component
+  It does not support icon replacement or icon hiding
+
+  Additions / changes:
+  - added useTranslation
+  - added hideIcon prop
+  - added iconReplacement prop (icon is rendered if this is set and external is false)
+  - added aria-label as prop
+  - default size is "M" instead of "S"
+  - icon size is "xs" instead of "s" when size is "M"
+  - getTextFromReactChildren -function is copied from own NewWindowLinkWithChildrenAsTitle.tsx.
+    The HDS version is not importable
+*/
+import React from 'react';
+import 'hds-core';
+import { IconLinkExternal } from 'hds-react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import styles from './Link.module.css';
+
+export type LinkProps = Omit<
+  React.ComponentPropsWithoutRef<'a'>,
+  | 'target'
+  | 'href'
+  | 'onPointerEnterCapture'
+  | 'onPointerLeaveCapture'
+  | 'aria-label'
+> & {
+  /**
+   * Link content
+   */
+  children: string;
+  /**
+   * Boolean indicating whether visited styles of the link are applied
+   */
+  disableVisitedStyles?: boolean;
+  /**
+   * Boolean indicating whether the link will lead user to external domain.
+   */
+  external?: boolean;
+  /**
+   * Hypertext Reference of the link.
+   */
+  href: string;
+  /**
+   * Element placed on the left side of the link text
+   */
+  iconLeft?: React.ReactNode;
+  /**
+   * Boolean indicating whether the link will open in new tab or not.
+   */
+  openInNewTab?: boolean;
+  /**
+   * Aria label for opening link in a new tab
+   */
+  openInNewTabAriaLabel?: string;
+  /**
+   * Aria label for opening link in an external domain
+   */
+  openInExternalDomainAriaLabel?: string;
+  /**
+   * Size of the link
+   */
+  size?: 'S' | 'M' | 'L';
+  /**
+   * Additional styles
+   */
+  style?: React.CSSProperties;
+  /**
+   * hideIcon
+   */
+  hideIcon?: boolean;
+  /**
+   * Element placed on the left side of the link text
+   */
+  iconReplacement?: React.ElementType;
+  /**
+   * The computed aria-label can be overridden with this prop
+   */
+  'aria-label'?: string;
```

Comment:
The "aria-label" is a HTML-attribute, and "ariaLabel" would need to be converted back to "aria-label" when setting it to the element. Now it can be set by using just "{...rest}" in line 150.

But anyway "aria-label" is already included in the element props, because it inherits from "a"-element. So I just removed it from the file.